homeassistant/components/statistics/services.yaml

```diff
@@ -1,3 +1,39 @@
 reload:
   name: Reload
   description: Reload all statistics entities.
```

Comment:
@emontnemery I was not sure whether this one can be deleted

Reply:
This is to give a helpful description for the reload service which is registered by calling `async_setup_reload_service`

Reply:
Think you misunderstood me. I question whether this "reload" service is actually needed. I don't have to touch it. Just wondering whether to deprecate it!?

Reply:
You mean deprecate reload in favor of the `reset_buffer` added by this PR?

Reply:
Yes. It gives more control and I don't think `reload` has a sharp use case.

Reply:
The purpose of the `reload` service is to allow adding or modify statistics sensors and make the new configuration take effect without restarting Home Assistant.